client/src/Components/ProductCard.tsx

```diff
@@ -10,25 +10,23 @@ type ProductCardProps = {
   purchaseHistory: number;
 };
 
+//todo: make the counter button a stable size
+
```

Comment:
Do we need a cleanup here or is this something that needs to be implemented?

Reply:
This was something that needed to be implemented, thanks for catching this. It's now implemented.